### ISSUE TITLE
Bug: no aria-label for close icon button in lightbox header

### ISSUE THREAD

**Issue by jonasch** [sbb-design-systems/sbb-angular#1152]
**Describe the bug**
We got feedback in terms of accessibility for our application. One thing was that the close button in the lightbox header does not have an alt text or aria label. Since it is necessary for icon-only buttons, we should have a configuration property or a default text for that, e.g. "close the dialog"

**To Reproduce**
1. Go to ['Ligbtox Examples page'](https://angular.app.sbb.ch/angular/components/lightbox/examples)
2. Click on 'Open Lightbox from component'
3. Inspect close button in upper right corner with accessibility viewer or screen reader
4. See no aria label or description

**Expected behavior**
Should have a default aria label or a config to set this

**Screenshots**
![image](https://user-images.githubusercontent.com/9671803/154479236-7c0b62ff-178a-4003-9e6b-46f9b41d1240.png)
